src/Themes/InputField.css

```diff
@@ -0,0 +1,13 @@
+.inputfield{
```

Comment:
Also add a focus state for inputs.
```suggestion
.input-field{
```